packages/AutoTDD-Core.package/ATDDTestingImage.class/class/start.st

```diff
@@ -0,0 +1,6 @@
+initialization
+start
+
+	FileDirectory default forceNewFileNamed: 'startServerScript.st' do: [ :str | str nextPutAll: 'ATDDHeadlessImage startServer.'].
```

Comment:
```suggestion
	FileDirectory default forceNewFileNamed: 'startServerScript.st' do: [ :str | str nextPutAll: 'ATDDTestingImage startServer.'].
```

Reply:
I think you forgot to rename the class here, probably didn't do it automatically.

Reply:
Good point :+1: That would have been bad :D